services/replicator/metadataReconciler.go

```diff
@@ -71,7 +71,7 @@ const (
 	runInterval                    = time.Duration(10 * time.Minute)
 	metadataListRequestPageSize    = 50
 	storeCallTimeout               = 10 * time.Second
-	extentMissingDurationThreshold = time.Duration(24 * time.Hour)
+	extentMissingDurationThreshold = time.Duration(1 * time.Hour)
```

Comment:
Good point. I have it but it seems not working. Will look at that first then switch over.